maven-plugin/tests/jt-250/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>jaxb-maven-plugin-tests-250</artifactId>
+    <parent>
+        <groupId>org.jvnet.jaxb</groupId>
+        <artifactId>jaxb-maven-plugin-tests</artifactId>
+        <version>2.0.4-SNAPSHOT</version>
+    </parent>
+    <name>JAXB Tools :: Maven Plugin :: Test [JAXB-TOOLS 250]</name>
```

Comment:
What do you think about using the <description>. . tag to explain what the test does? I like naming the tests after the ticket number, but might be handy to have the basic gist right there without having to hit the link.  obviously, for complex things, the ticket detail will have all the notes.